packages/mdc-textfield/index.js

```diff
@@ -249,6 +257,17 @@ class MDCTextfield extends MDCComponent {
           this.helptextElement.removeAttribute(name);
         }
       },
+      setHelptextValue: (value) => {
+        if (this.helptextElement) {
+          this.helptextElement.textContent = value;
+        }
+      },
+      getHelptextValue: () => {
+        if (!this.helptextElement) {
+          return;
+        }
+        return this.helptextElement.textContent.trim();
```

Comment:
What's the reason for adding `trim()`? I assume users would want to know if they have extra whitespace.

Reply:
Good point, I was trying to ensure that the returned value was close to what the user expected but that sounds like a false assumption on my part.

For example:
```
<p class="mdc-text-field-helptext">
  Hello
</p>
```
would have a `textContent` of `"  Hello "` when I would expect it to be just `"Hello"`

That being said, I see now that's an incorrect assumption. I'll remove `.trim()`. Thanks!